src/sidebars/favourites/geometry-service.js

```diff
@@ -77,12 +77,6 @@
     }
 
     class GeometryService {
-        windowsIntersect(openFinWindow, _window) {
-            var nativeWindow = openFinWindow.getNativeWindow();
-
-            return intersectHelper(getWindowPosition(nativeWindow), getWindowPosition(_window));
-        }
```

Comment:
Similarly does this snowball? Are `intersectHelper` and `getWindowPosition` still used?